### PR TITLE
build: bump year

### PR DESCRIPTION
**Pull request by CaerusKaru**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/static-query-migration.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Static Query Migration Guide
 ​
-**Important note for library authors: This migration is especially crucial for library authors to facilitate their users upgrading to version 9 when it becomes available (approx Oct 2019).**
+**Important note for library authors: This migration is especially crucial for library authors to facilitate their users upgrading to version 9 when it becomes available (approx Jan 2020).**
```
Comment (StephenFluin) [inline aio/content/guide/static-query-migration.md]: Can you just remove the date?

**Review by StephenFluin — CHANGES_REQUESTED** [review]

**Review by StephenFluin — APPROVED** [review]
LGTM. I'll defer to others regarding overall legal requirements for changing year.

**Review by filipesilva — APPROVED** [review]

**Review by petebacondarwin — APPROVED** [review]

**Review by gkalpak — APPROVED** [review]
Could you also update [aio/src/404-body.html][1], which seems to be stuck in 2018 :grin:

[1]: https://github.com/angular/angular/blob/5b864ede1/aio/src/404-body.html#L45

**Review by devversion — APPROVED** [review]
LGTM. FYI: It looks like there is a second PR for bumping the year: https://github.com/angular/angular/pull/34604

**Comment by CaerusKaru** [comment]
@gkalpak All set :+1:

**Review by IgorMinar — APPROVED** [review]
thanks!

**Comment by IgorMinar** [comment]
merge-assistance: global approval

**Pull request closed without merging**